package-lock.json

```diff
@@ -3772,15 +3772,6 @@
         "sha.js": "^2.4.8"
```

Comment:
we use `yarn` instead of `package-lock.json`, just remove `package-lock.json`, run `yarn install` and add `yarn.lock` to the repo